src/test/java/com/microsoft/sqlserver/jdbc/bulkCopy/BulkCopyCSVTest.java

```diff
@@ -446,6 +450,114 @@ public void testCSV2400() {
         }
     }
 
+   /**
+     * Test to perform bulk copy with a computed column as the last column in the table.
+     */
+    @Test
+    @DisplayName("Test bulk copy with computed column as last column")
+    public void testBulkCopyWithComputedColumnAsLastColumn() {
+        String tableName = AbstractSQLGenerator.escapeIdentifier(RandomUtil.getIdentifier("BulkEscape"));
+        String fileName = filePath + computeColumnCsvFile;
+
+        assertDoesNotThrow(() -> {
```

Comment:
this is unnecessary and to keep consistent look like the other tests, can we remove?